### ISSUE TITLE
Accessibility: small arrow in the bottom right corner of the Find dialog is not accessible with NVDA

### ISSUE THREAD

**Issue by mltony** [microsoft/vscode#204864]
Type: <b>Bug</b>

I am blind and use VSCode with NVDA screenreader. Previously I've been able to adjust find settings within `Control+F` find dialog just by tabbing to them. I am talking about settings like regexp search and search for partial word. However after a recent update I haven't been able to do so. AI [suggests](https://www.phind.com/search?cache=dqlmqf4yt5knlhngnqmeyy6f) that there is supposed to be a "small arrow in the bottom right corner of the Find dialog", which reveals all those extra options. However NVDA doesn't see that small arrow, so I suspect it is not being made accessible. If AI didn't hallucinate and said button is indeed visually present - could you please make it accessible for screenreader users?

VS Code version: Code 1.86.1 (31c37ee8f63491495ac49e43b8544550fbae4533, 2024-02-07T09:08:20.941Z)
OS version: Windows_NT x64 10.0.22621
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 4700U with Radeon Graphics          (8 x 1996)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.37GB (2.60GB free)|
|Process Argv|--crash-reporter-id 9a96d9f3-a840-44d8-9d62-0bac44c9d4b9|
|Screen Reader|yes|
|VM|67%|
</details><details><summary>Extensions (17)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.4.2
doxdocgen|csc|1.4.0
better-cpp-syntax|jef|1.17.2
goto-next-previous-member|mis|0.0.6
debugpy|ms-|2024.0.0
python|ms-|2024.0.1
vscode-pylance|ms-|2024.2.1
jupyter|ms-|2024.1.1
jupyter-keymap|ms-|1.1.2
jupyter-renderers|ms-|1.0.17
vscode-jupyter-cell-tags|ms-|0.1.8
vscode-jupyter-slideshow|ms-|0.1.5
cmake-tools|ms-|1.16.32
cpptools|ms-|1.18.5
cpptools-extension-pack|ms-|1.3.0
indent-nav|Ton|1.0.3
cmake|twx|0.0.17

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscoreces:30445986
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
vscaat:30438848
c4g48928:30535728
azure-dev_surveyone:30548225
2i9eh265:30646982
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
accentitlementsc:30887149
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
pydisww1:30959800
3ef8e399:30949928
e3gdj431:30958358

```

</details>

**Comment by mltony** [comment]
Never mind, AI hallucinated and I figured out that I had too many editor groups open. This is not a bug.
